portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +29,111 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+//Add random quote
+function addRandomQuote() {
+  const B99Quotes = [
+      '"If I die, turn my tweets into a book." -Gina Linetti', 
+      '"The English language can not fully capture the depth and complexity of my thoughts, so I’m incorporating emojis into my speech to better express myself. Winky face." - Ginna Linetti', 
+      '"I have zero interest in food. If it were feasible, my diet would consist entirely of flavorless beige smoothies containing all the nutrients required by the human animal." - Captain Ray Holt', 
+      '"I only have one dream a year, always on Tax Day. In it, I must file an extension. So, yes, it is best not to have dreams." - Captian Ray Holt'];
+
+  // Pick a random greeting.
+  const quote = B99Quotes[Math.floor(Math.random() * B99Quotes.length)];
+
+  // Add it to the page.
+  document.getElementById('quote-container1').innerText = quote;
+  
+}
+
+
+async function getB99QuoteServlet() {
+    //b99 quotes but from servlet instead of just js.
+    const response = await fetch('/b99');
+    console.log('Fetching a random quote.');
+    const quote = await response.text();
+    console.log('Adding quote to dom: ' + quote);
+    document.getElementById('quote-container2').innerText = quote;
+
+}
+
+/** 
```

Comment:
Get into the habit of not checking in any commented out code.

It makes it harder for people to work with your code – they don't have the context on whether this code is necessary, is it outdated, is it supposed to be uncommented at some point in the future?

It also makes it easier for you down the road if you eventually forget why you left some code commented out.